mico-core/src/main/java/io/github/ust/mico/core/dto/MicoApplicationStatusDTO.java

```diff
@@ -46,4 +45,24 @@
      */
     private List<MicoServiceStatusDTO> serviceStatus = new ArrayList<>();
 
+    /**
+     * Number of {@link MicoService}s belonging to a {@link io.github.ust.mico.core.model.MicoApplication}
```

Comment:
`{@link MicoApplication}` is enough with the right import. Same applies to the following JavaDoc comments.

Reply:
Adapted package imports for all JavaDoc comments in DTO classes.